### ISSUE TITLE
Role of Trust Anchors

### ISSUE THREAD

**Issue by hannestschofenig** [ietf-wg-scitt/draft-ietf-scitt-architecture#270]
_Editors note: updated with a link to the referenced text:_ 

>    [Transparency Services MUST also maintain a list of trust anchors, which SHOULD be used by Relying Parties to authenticate Issuers, and which MAY be included in a Registration Policy statement.](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/blob/268-figure-1-description/draft-ietf-scitt-architecture.md?plain=1#L407)

I don't understand the SHOULD.

You have to be in possession of trust anchors to verify a digital signature. It is not an optional feature.

**Comment by aj-stein-nist** [comment]
> I don't understand the SHOULD.
> 
> You have to be in possession of trust anchors to verify a digital signature. It is not an optional feature.

It would seem this is correct contingent on your interpretation of [the requirements of TS registration steps, especially](https://ietf-wg-scitt.github.io/draft-ietf-scitt-architecture/draft-ietf-scitt-architecture.html#name-registration), particularly points 2 and 3, so good point.

**Comment by robinbryce** [comment]
It's not the use of the anchor in verifying that is optional. It is the use of issuer verification as a gate to registration that is the problem. 

For ts implementations that naturaly have RBAC at the edge, the "spam" problem is not really a problem.

Statements about statements seem to me to be a better way to deal with use cases where strong issuer verification is necessary. Registration time checks can only ever be "point in time"

**Comment by SteveLasker** [comment]
My read of the Issue and the current state of the document is:

- This section is under [Registration Policies](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/blob/268-figure-1-description/draft-ietf-scitt-architecture.md?plain=1#L401), which implies the MUST refers to enabling the registration process. Currently, a registration can not be completed unless the issuer is verified, which can't occur without trust anchors.
The SHOULD refers to verifiers which are not part of the Registration Policy. I think their inclusion in this section is confusing. 

See #304 for a proposal.
